### DISCUSSION TITLE
Is there ways to trigger dangerjs without new commit?

### DISCUSSION THREAD

**Discussion by michael-land** [danger/danger-js#1247] [Q&A] [answered]
Hi, I'm new here.

I leave assignees empty on purpose when creating a PR. Then, I assigned myself after danger.js checks.

any ideas how to trigger re-check without push another commit?

<img width="600" alt="image" src="https://user-images.githubusercontent.com/33362998/155825642-c3bc4eee-63f7-4fcb-9884-371f7cf43ec0.png">

**Comment by fbartho** [comment, marked as answer]
What is your GitHub Actions workflow file triggered on?

We trigger danger-js in our repo on open, synchronize, and labeled, but you can probably trigger it on assign?
